packages/api/src/task/scheduler.ts

```diff
@@ -202,6 +202,22 @@ export class TaskScheduler {
           });
         }
         break;
+      case "export-data":
+        if (task.params["export-data"].type === "attestation") {
```

Comment:
I believe you need to handle the failure case as well (IIRC a `failTask` function in this scheduler) to update the status to `failed` etc. Otherwise it'd be pending forever

Reply:
Good point! Updated in 58a5a60c263cff7638266e9da03a382c2510dcce